src/illuminatio/test_orchestrator.py

```diff
@@ -301,44 +323,47 @@ def collect_results(self, api: k8s.client.CoreV1Api):
                 if api_exception.reason == "Not Found":
                     pass
                 else:
-                    raise(api_exception)
-            logger.debug("Map names: " + str([m.metadata.name for m in result_config_maps]))
-            logger.debug("Expected names: " + str(expected_result_map_names))
+                    raise api_exception
+            self.logger.debug("Map names: " + str([m.metadata.name for m in result_config_maps]))
+            self.logger.debug("Expected names: " + str(expected_result_map_names))
             time.sleep(2)
         yamls = [yaml.safe_load(c.data["results"]) for c in result_config_maps]
-        logger.debug("Found following yamls in result config maps:" + str(yamls))
+        self.logger.debug("Found following yamls in result config maps:" + str(yamls))
         times = {c.metadata.name: yaml.safe_load(c.data["runtimes"])
                  for c in result_config_maps if "runtimes" in c.data}
         return {k: v for yam in [y.items() for y in yamls] for k, v in yam}, times
 
-    def createDaemonset(self, daemon_set_name, service_account_name, config_map_name, api):
+    def create_daemonset(self, daemon_set_name, service_account_name, config_map_name, api):
+        """
+        Creates a DaemonSet on basis of the project's manifest files
+        """
         # load suitable manifest
         container_runtime_version = get_container_runtime()
         if container_runtime_version.startswith("docker"):
-            daemon_set_dict = getManifest("docker-daemonset.yaml")
+            daemon_set_dict = get_manifest("docker-daemonset.yaml")
         elif container_runtime_version.startswith("containerd"):
-            daemon_set_dict = getManifest("containerd-daemonset.yaml")
+            daemon_set_dict = get_manifest("containerd-daemonset.yaml")
         else:
             raise NotImplementedError("Unsupported container runtime: %s" % container_runtime_version)
 
         # adapt non-static values
         daemon_set_dict["metadata"]["name"] = daemon_set_name
         daemon_set_dict["metadata"]["namespace"] = PROJECT_NAMESPACE
-        daemon_set_dict["spec"]["template"]["spec"]["containers"][0]["image"] = self.runner_image
+        daemon_set_dict["spec"]["template"]["spec"]["containers"][0]["image"] = self.docker_image["runner"]
```

Comment:
Please rename `docker_image ` also nitpick -> this should actually be called `docker_images` because the data structure contains more than one image.